x-pack/solutions/observability/plugins/observability_agent_builder/server/tools/get_change_points/get_log_change_points.ts

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AggregationsAggregationContainer } from '@elastic/elasticsearch/lib/api/types';
```

Comment:
Since you started on this PR we have settled on a single tool per folder. Also, add a readme to each of the tools with examples on how to call it